### PR TITLE
Add posibility to set login request timeout.

### PR DESCRIPTION
**Pull request by stepek**

### CODE REVIEW

File: app/config.json.sample [hunk 1]
```diff
@@ -17,5 +17,6 @@
     "addingRelationDialog": [
         "Pet"
     ],
-    "pageLimit": 25
+    "pageLimit": 25,
+    "loginTimeout": 30000
```
Comment (thealmightyone) [inline app/config.json.sample]: so you have 30 seconds to choose a tenant when you first input auth data?

Comment (stepek) [inline app/config.json.sample]: No, this is a login request timeout.

**Comment by stepek** [comment]
Fixed test and name of json field.

**Comment by thayashi** [comment]
LGTM

**Pull request merged**